regolith/filter.go

```diff
@@ -12,6 +12,41 @@ type Filter struct {
 	Settings    map[string]interface{} `json:"settings,omitempty"`
 }
 
+type RunContext struct {
+	AbsoluteLocation string
+	Config           *Config
+	Profile          string
+	Parent           *RunContext
+}
+
+type ProfileFilter struct {
```

Comment:
I would move the `ProfileFilter` into its own file called *filter_profile.go* to follow the convention of the other filters.

Reply:
Done.